src/components/molecules/toast.tsx

```diff
@@ -114,6 +116,13 @@ export const Toast = () => {
 								<p className={titleStyle(type)}>{title}</p>
 
 								{message}
+								<Button
+									className="cursor-pointer gap-1.5 p-0 font-medium text-error underline"
+									onClick={() => toggleLogger(true)}
+								>
+									Show more
+									<ExternalLinkIcon className="size-3.5 fill-error duration-200" />
+								</Button>
```

Comment:
We should show it only when the toast type is `error`

Reply:
fixed